SECURITY.md

```diff
@@ -0,0 +1,55 @@
+# Security Policy
+
+## Supported Versions
+
+We commit to publishing security updates for the version of Flutter currently
+on the `stable` branch.
+
+## Reporting a Vulnerability
+
+To report a vulnerability, please e-mail `security@flutter.dev` with a description of the issue,
+the steps you took to create the issue, affected versions, and if known, mitigations for the issue.
+
+We should reply within three working days, probably much sooner.
+
+We use GitHub's security advisory feature to track open security issues. You should expect
+a close collaboration as we work to resolve the issue you have resolved. Please reach out to
+`security@flutter.dev` again if you do not receive prompt attention and regular updates. You
+may also reach out to the team via our public [Chat] channels; however, please make sure to
+e-mail `security@flutter.dev` when reporting an issue.
+
+## Process
+
+This section describes the process used by the Flutter team when handling vulnerability reports.
+
+Vulnerability reports are received via the security@flutter.dev e-mail alias. Certain team members
+who have been designated the "vulnerability management team" receive these e-mails. When receiving
+such an e-mail, you must:
```

Comment:
s/you/they/ .  Otherwise it sounds like the person sending the email must do those steps...

Reply:
.. and similar changes below

Reply:
fixed